src/grasshopper/lib/fixtures/__init__.py

```diff
@@ -476,3 +512,14 @@ def fetch_value_from_multiple_sources(sources, key):
     for source in sources:
         value = value or source.get(key)
     return value
+
+
+def type_check_list_of_strs(list_of_strs):
+    """Return True if list of strings or [], false if anything else."""
+    check_passed = False
+    if type(list_of_strs) == list:
+        all_strs = True
+        for s in list_of_strs:
```

Comment:
if `list_of_strs = []`, I think this will return False, even though the docstring says otherwise

Reply:
This works. An empty list is still a list, so the first if is True. Then the for loop will do 0 iterations, because the list is empty. This means that what is returned comes from line 521, True.

I think there is a unit test for this case, if not I will add one.

Reply:
checked and there is a unit test for this case already